app/models/gi_bill_feedback.rb

```diff
@@ -75,17 +59,14 @@ def transform_form
     transformed.merge!(get_user_details)
 
     transformed['education_details'].tap do |education_details|
-      next if education_details.blank?
```

Comment:
why is this deleted?

Reply:
since `location_code` is no longer an option, `education_details` is required/popluated on the frontend whether via school search or manual entry.
[Required in the schema too.](https://github.com/department-of-veterans-affairs/vets-json-schema/blob/master/src/schemas/FEEDBACK-TOOL/schema.js#L134)